resources/public/js/audit-recovery.js

```diff
@@ -2,8 +2,11 @@ $(document).ready(function() {
   $("#audit_recovery_steps").hide();
   showLoader();
   var apiUri = "/api/audit-recovery/" + currentPage();
-  if (auditCluster()) {
-    apiUri = "/api/audit-recovery/cluster/" + auditCluster() + "/" + currentPage();
+  if (clusterName()) {
+    apiUri = "/api/audit-recovery/cluster/" + clusterName() + "/" + currentPage();
+  }
```

Comment:
Why not `else if`? Name & alias cannot exist at the same time. State it in the code.

Reply:
just to follow convention, current code is like this in multiple places (e.g. `audit-failure-detection.js`, can change on a follow up MR if wanted